.cursor/rules/django-python.mdc

```diff
@@ -0,0 +1,105 @@
+---
+description: Rules for writing Python with Django (most PostHog backend code)
+globs: ee/*.py, posthog/*.py
```

Comment:
I kinda just picked some folders that seemed to be mostly Django-related, but I know we have a bunch of HogQL++ in `posthog/` as well, so maybe these examples don't apply.  I don't know which directories to explicitly include – maybe I should just go with `posthog/api`, `posthog/models`?